dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/support/eager/EagerThreadPool.java

```diff
@@ -51,7 +51,7 @@ public Executor getExecutor(URL url) {
         int alive = url.getParameter(ALIVE_KEY, DEFAULT_ALIVE);
 
         // init queue and executor
-        TaskQueue<Runnable> taskQueue = new TaskQueue<Runnable>(queues <= 0 ? 1 : queues);
+        TaskQueue<Runnable> taskQueue = new TaskQueue<Runnable>(Math.max(1, queues));
```

Comment:
Both of two lines got the same meaning, I believe it's unnecessary to change this.